pkgs/tools/networking/ogen/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  pkgs,
+  lib,
+}:
+pkgs.buildGoModule rec {
+  pname = "ogen";
+  version = "1.0.0";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "ogen-go";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-wIcxICfBY5YHVqKHtHd2wzcaDmkzgp7eD6zEvnsRaIU=";
+  };
+
+  vendorHash = "sha256-EL8FcAnDMekHBIDRdGqQO4JNESjGC7MQ5aUcDkRqrbg=";
+  doCheck = false;
+
+  subPackages = ["cmd/ogen"];
+
+  meta = with lib; {
+    description = "OpenAPI v3 code generator for go";
+    homepage = "https://github.com/ogen-go/ogen";
+  };
```

Comment:
Suggest you add the following meta attributes:

- [changelog](https://nixos.org/manual/nixpkgs/stable/#var-meta-changelog),
- [license](https://nixos.org/manual/nixpkgs/stable/#var-meta-license),
- [maintainers](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers),
- [mainProgram](https://nixos.org/manual/nixpkgs/stable/#var-meta-mainProgram) and
- [platforms](https://nixos.org/manual/nixpkgs/stable/#var-meta-platforms).